testing/integration/otel_test.go

```diff
@@ -1042,23 +1043,32 @@ func TestOtelFilestreamInput(t *testing.T) {
         file_identity.native: ~
 outputs:
   default:
+    type: elasticsearch
+    hosts: [{{.ESEndpoint}}]
+    api_key: "{{.ESApiKeyEncoded}}"
```

Comment:
These key encoding shenanigans will be unnecessary once we incorporate https://github.com/elastic/beats/pull/44031.

Reply:
FYI: https://github.com/elastic/beats/pull/44031 has been merged.

Reply:
I'll fix the tests in a follow-up.